crates/services/sync/src/sync.rs

```diff
@@ -53,6 +60,8 @@ impl SyncHeights {
     /// Sync the state from the height stream.
     /// This stream never blocks or errors.
     pub(crate) async fn sync(&mut self) -> Option<()> {
+        let start = ServiceMetrics::instant();
```

Comment:
It measures busy and idle time together, we want to know about both independently.